sys/stdvga.c

```diff
@@ -0,0 +1,165 @@
+#include <pci.h>
+#include <vga.h>
+#include <stdc.h>
+#include <malloc.h>
+
+#define VGA_PALETTE_SIZE (256 * 3)
+
+typedef struct stdvga_device {
+  pci_device_t *pci_device;
+  resource_t mem;
```

Comment:
Please keep these as pointers. Once I consolidate bus interface there will be call similar to [bus_alloc_resource_any](https://www.freebsd.org/cgi/man.cgi?query=bus_alloc_resource_any&sektion=9) to reserve a BAR.